store/modules/stimulation/getters.js

```diff
@@ -22,7 +20,17 @@ export default {
 
     if (!state.edit_mode.status) {
       const letter = get_protocol_editor_letter(protocol_list);
-      const color = get_protocol_editor_color(protocol_list);
+      // loop through again once protocol list becomes greater than 24
+      let num_of_loops = 0;
+      let num = 26;
+      while (num < protocol_list.length) {
+        num += 26;
+        num_of_loops += 1;
+      }
+
+      const offset = 26 * num_of_loops + 1;
+      const color = COLOR_PALETTE[protocol_list.length - offset];
```

Comment:
You can achieve wrapping back around to the start by just using the modulo operator
```
> nums = Array.from({length: 30}, (x, i) => i);
[
   0,  1,  2,  3,  4,  5,  6,  7,  8,
   9, 10, 11, 12, 13, 14, 15, 16, 17,
  18, 19, 20, 21, 22, 23, 24, 25, 26,
  27, 28, 29
]
> nums.map(n => n % 26)
[
   0,  1,  2,  3,  4,  5,  6,  7,  8,
   9, 10, 11, 12, 13, 14, 15, 16, 17,
  18, 19, 20, 21, 22, 23, 24, 25,  0,
   1,  2,  3
]
```